ldpd/ldp_zebra.c

```diff
@@ -153,18 +156,84 @@ kr_delete(struct kroute *kr)
 	return (zebra_send_mpls_labels(ZEBRA_MPLS_LABELS_DELETE, kr));
 }
 
+static int
```

Comment:
Does this function Belong in zclient.c to allow other, future, routing protocols to take advantage of it?  Should it be under zapi_route as well?  Instead of a specialized function?